src/alpaca.erl

```diff
@@ -411,4 +411,24 @@ throw_with_variables_test() ->
     ?assertThrow({not_equal, 1, 2}, M:assert_equal(1, 2)),
     code:delete(M).
 
+record_update_test() ->
+    Files = ["test_files/update_record.alp",
+             "test_files/use_update_record.alp"],
+    [M1, M2] = compile_and_load(Files, []),
+
+    ?assertEqual(#{'__struct__' => record, x => 5, b => 2}, M2:main(unit)),
```

Comment:
We could do that.  I've always just treated it as a throwaway value and not bothered to check.  I don't have particularly strong opinions on this but our code gen _does_ output an empty Erlang tuple.  I suppose this is more of an issue for Erlang calling Alpaca.

I'll change this particular occurrence to an empty tuple, happy to have the broader "do we crash on anything not an empty tuple" in a separate issue.